index.html

```diff
@@ -4800,19 +4801,18 @@ <h2 id="AudioBufferSourceNode">
             <p>
               An optional value in seconds where looping should end if the
               <code>loop</code> attribute is true. Its value is exclusive of
-              the content of the loop: the sample frames comprising the loop
-              run from the values <code>loopStart</code> to
-              <code>loopEnd-(1/sampleRate)</code>. Its default
-              <code>value</code> is 0, and it may usefully be set to any value
-              between 0 and the duration of the buffer. If <code>loopEnd</code>
-              is less than 0, looping will end at 0. If <code>loopEnd</code> is
-              greater than the duration of the buffer, looping will end at the
-              end of the buffer. This attribute is converted to an exact sample
-              frame offset within the buffer by multiplying by the buffer's
-              sample rate and rounding to the nearest integer value. Thus its
-              behavior is independent of the value of the <a href=
+              the content of the loop. Its default <code>value</code> is 0, and
+              it may usefully be set to any value between 0 and the duration of
+              the buffer. If <code>loopEnd</code> is less than or equal to 0,
+              or if <code>loopEnd</code> is greater than the duration of the
+              buffer, looping will end at the end of the buffer. The loop
+              endpoint within the buffer is determined by multiplying by the
```

Comment:
Multiply what by the buffer's sample rate?  Something is missing here. Plus, I think we don't really care anymore what frame it translates to.